tldr/cross_file_calls.py

```
@@ -3939,3 +3943,291 @@ def _build_php_call_graph(
+                            resolved = True
+                        else:
+                            key = (fq_module, call_target)
+                            if key in func_index:
+                                graph.add_edge(rel_path, caller_func, func_index[key], call_target)
+                                resolved = True
+                    if not resolved and call_target in name_index and len(name_index[call_target]) == 1:
+                        target_file, _ = name_index[call_target][0]
+                        graph.add_edge(rel_path, caller_func, target_file, call_target)
+
+                elif call_type == 'attr':
+                    if '.' in call_target:
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)